File: demo/app/app/src/main/java/com/vectordemo/domain/dto/http/request/UserTokenVerifyRequest.kt
package com.vectordemo.domain.dto.http.request

import com.google.gson.annotations.JsonAdapter
import com.vectordemo.utils.json.WireUserIdJsonDeserializer

data class UserTokenVerifyRequest(
    @JsonAdapter(WireUserIdJsonDeserializer::class)
    var userId: String? = null,
    var accessToken: String? = null
)

File: demo/flutter/flutternew/lib/domain/dto/auth/user_token_verify_request.dart
class UserTokenVerifyRequest {
  final int userId;
  final String accessToken;

  UserTokenVerifyRequest({
    required this.userId,
    required this.accessToken,
  });

  Map<String, dynamic> toJson() {
    return {
      'userId': userId,
      'accessToken': accessToken,
    };
  }
}
